lab-cortney/gulpfile.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const gulp = require('gulp');
+const mocha = require('gulp-mocha');
+const eslint = require('gulp-eslint');
+
+gulp.task('default', function() {
```

Comment:
While your watch task does perform both linting and testing I suggest you create a separate task for running your tests and then specify the default task to run both the linter and tests task instead of setting the default task to be the test task.  Just a suggestion.